classifides/ads/models.py

```diff
@@ -1,3 +1,61 @@
 from django.db import models
+from django.conf import settings
+from taggit.managers import TaggableManager
 
-# Create your models here.
+User = settings.AUTH_USER_MODEL
+
+
+class Category(models.Model):
+    CATEGORY_TYPE_CHOICES = [
+        ('job', 'Job'),
+        ('gig', 'Gig'),
+        ('rental', 'Rental'),
+        ('for_sale', 'For Sale'),
+        ('service', 'Service'),
+        ('event', 'Event'),
+        ('class', 'Class'),
+    ]
+    
+    name = models.CharField(max_length=100)
+    description = models.TextField(blank=True, null=True)
+    category_type = models.CharField(max_length=20, choices=CATEGORY_TYPE_CHOICES)
+
+    def __str__(self):
+        return self.name
+
+class Ad(models.Model):
+    category = models.ForeignKey(Category, on_delete=models.CASCADE)
+    title = models.CharField(max_length=200)
+    image = models.ImageField(upload_to='ads/images/', blank=True, null=True)
```

Comment:
An ad can have multiple images. Create a separate model for `AdImage`   . AdImage model will have following fields:
- image
- uploaded by
- ad foreign key
-

Reply:
ok